Doc/library/shutil.rst

```diff
@@ -160,7 +160,8 @@ Directory and files operations
    Copies the file *src* to the file or directory *dst*.  *src* and *dst*
    should be :term:`path-like objects <path-like object>` or strings.  If
    *dst* specifies a directory, the file will be copied into *dst* using the
-   base filename from *src*.  Returns the path to the newly created file.
+   base filename from *src*. If *dst* already exists, it will be replaced.
```

Comment:
```suggestion
   base filename from *src*. If *dst* specifies a file and the file already exists, it will be replaced.
```
If it's a directory that already exists, it's not replaced. (Feel free to use different wording.)